command/start.go

```diff
@@ -89,6 +90,13 @@ func (c *startCommand) startFlags() *flag.FlagSet {
 	flags.BoolVar(&autocompleteUninstall, flagAutocompleteUninstall, false, "Uninstall the autocomplete")
 	c.autocompleteUninstall = &autocompleteUninstall
 
+	// Sets that we are using start as the default command. Not printed with -h, -help
+	flags.BoolVar(&isDefault, flagDefault, false,
+		"Use only when running CTS without the start command. "+
+			"\n\t\tSignals any different actions CTS should perform when "+
+			"\n\t\tthis command is used as the default.")
```

Comment:
From the flag description, I initially though this PR was adding a new flag for users to set (it's not, I think?). What are your thoughts on language something like, this flag is set automatically by CTS when starting up without the start command....

Reply:
I got a little flip-flopped understanding "default" - my intuition was the `start` command was the default (this is what we want users to do) but if I understand correctly, no `start` cmd is the default.

I was wondering what your thoughts are on renaming the flag and related variables to maybe `-deprecated` or `-deprecated-start-up`? I know this work will be removed in a few releases so want to be mindful of not taking up much of your time. I think having "deprecated" in the language will make it easy and clear what to clean up.

Reply:
yeah, my reasoning for using `default` was because if you don't include a command, it's using the `start` command as default, and this signals to the command that it's using `start` as the default command, which changes some of the behaviour...

That might be unclear though, so I'm okay with deprecated-start-up?

Reply:
Ah interesting. Thanks for sharing your thought process. That makes sense. Yea, it looks like default can be interpreted different ways. `deprecated-start-up` works for me if it makes sense to you as well